pkg/controller/schedule_controller_test.go

```diff
@@ -62,6 +62,10 @@ func TestReconcileOfSchedule(t *testing.T) {
 			name:     "schedule with phase FailedValidation triggers no backup",
 			schedule: newScheduleBuilder(velerov1api.SchedulePhaseFailedValidation).Result(),
 		},
+		{
+			name:     "paused schedule triggers no backup",
+			schedule: newScheduleBuilder(velerov1api.SchedulePhase("")).Result(),
```

Comment:
The `paused` is not set in the test case.

Reply:
The pause cannot be tested easily in UT after moving the check logic into pridicate, I have remove this line and we'll cover the test in the E2E